azure-pipelines.yml

```diff
@@ -0,0 +1,94 @@
+pool:
+  name: Hosted VS2017
+
+variables:
+  - name: "Build.ArtifactStagingDirectory"
+    value: "Artifacts/"
+  # The following variables are defined as settable at queue time, so they cannot be included here
+  # Branch.CdfProvider: dnn
+  # Branch.CkEditor: development
+  # CakeTarget: BuildAll
+  # ReleaseMode: Beta
+  # RunTests: True
+
+trigger:
+  batch: true
+  branches:
+    include: '*'
+
+pr:
+  autoCancel: true
+  branches:
+    include: '*'

```

Comment:
Same comment as above

Reply:
@bdukes 
Following up to https://github.com/dnnsoftware/Dnn.Platform/pull/4327#discussion_r529018902

Currently the way the PR trigger is written it will start a build for any pull request to any branch. I am not sure if this is the best decision as there are branches that we may not want to auto-build against. This may be an incorrect assumption for me to make, but has been a best practice for projects I have worked on. 

Perhaps we adjust the PR workflow to only build against the main branches of the repo using the following yaml

```
pr:
  branches:
    include:
    - develop
    - master
    - future/10
```